inc/admin/namespace.php

```diff
@@ -52,7 +54,16 @@ function update_catalog() {
 	} else {
 		delete_blog_option( $blog_id, \Aldine\Admin\BLOG_OPTION );
 		update_site_meta( $blog_id, BookDataCollector::IN_CATALOG, 0 );
+		// Exclude book when network option book directory non-catalog exclude is enabled
+		$option = get_site_option( pressbooks_sharingandprivacy_options, [], true );
```

Comment:
I think should be cast to string the `pressbooks_sharingandprivacy_options` parameters

Reply:
revised